src/main/java/org/opentripplanner/routing/algorithm/raptor/transit/mappers/TripPatternForDateMapper.java

```diff
@@ -44,6 +51,13 @@
     this.newTripPatternForOld = newTripPatternForOld;
   }
```

Comment:
Using the `Collection.unmodifiableMap(...)` would be good idea here, then it becomes clear that these are READ_ONLY.